nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/NarUnpacker.java

```diff
@@ -264,7 +263,7 @@ private static void unpack(final File nar, final File workingDirectory, final by
                 String name = jarEntry.getName();
                 File f = new File(workingDirectory, name);
                 if (jarEntry.isDirectory()) {
-                    FileUtils.ensureDirectoryExistAndCanAccess(f);
+                    FileUtils.ensureDirectoryExistAndCanReadAndWrite(f);
```

Comment:
this looks like a change fixing a bug whereby we checked for simple access instead of ability to write files.  Can we link this to an existing JIRA or expand the scope of NIFI-5376 so we dont lose track of this.

Reply:
The deprecated method was one line simply calling the method with which I replaced it. There is no behavioral change. I imagine the original fix changed behavior but did not update all calling references.